virtcontainers/sandbox.go

```diff
@@ -1043,6 +1043,17 @@ func (s *Sandbox) stopVM() error {
 	}
 
 	s.Logger().Info("Stopping VM")
+
+	defer func() {
+		// if failed to stop sandbox in vm, we should check whether vm still running
+		if err != nil {
+			if running, _ := utils.IsProcRunning(s.hypervisor.pid()); !running {
```

Comment:
@Ace-Tang What if you performed this check first thing in stopVM to avoid grpc timeout.

Reply:
I make it check first.